earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -14,13 +14,21 @@ sudo yum install -y epel-release
 ### RHEL 7
 
 ```bash
+# Setup subscriptions for RHEL 7
```

Comment:
Minor: it would be nice to have some indication that this only applies to non-AWS RHEL 7 instances. You could include "non-AWS" in this comment, or maybe put the AWS version first and then change the comment to something like "For standard RHEL 7".